tests/model_fields/test_filefield.py

```diff
@@ -63,6 +64,15 @@ def test_refresh_from_db(self):
         d.refresh_from_db()
         self.assertIs(d.myfile.instance, d)
 
+    @unittest.skipIf(sys.platform == 'win32', "Crashes with OSError on Windows.")
```

Comment:
`test_save_without_name()` crashes on Windows with `OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Jenkins\\workspace\\pull-requests-windows\\database\\sqlite3\\label\\windows\\python\\Python39\\unused\\C:'`, however it did the same before 0b79eb36915d178aef5c6a7bbce71b1e76d376d3. It's not sth that we should change or improve in this patch.